### PR TITLE
Minor optimizations

### PR DESCRIPTION
**Pull request by anderslindho**
I stumbled upon your project and glanced at your source code, and noted some things that I thought were quickly added optimizations; I think all improve both performance and readability. See if you agree.

It's only really some list comprehension, and updating the imports.

### CODE REVIEW

**Comment by sylhare** [comment]
Awesome!

File: gitlab_stats/utils.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 import datetime
 import os
 
-import gitlab_stats
```
Comment (sylhare) [inline gitlab_stats/utils.py:5]: I think it is used in the file Line 87 for the CLI report.

**Review by sylhare — CHANGES_REQUESTED** [review]
I think removing the `import` made the pipeline crash https://travis-ci.org/github/sylhare/gitlab_stats/jobs/767918799

Otherwise I love the optimization it looks neat 👍

File: gitlab_stats/utils.py (hunk 1 above)
Comment (anderslindho) [reply gitlab_stats/utils.py:5]: Oops, my bad!

**Pull request merged**